src/scicookie/hooks/post_gen_project.py

```diff
@@ -262,7 +265,13 @@ def add_binding_source_files():
             os.makedir(src_system_dir)
             shutil.move(build_system_dir / "lib.rs", src_system_dir)
     else:
-        pass   
+        pass
+
+
+def clean_up_mypy():
+    if not USE_MYPY:
+        remove_package_file("py.typed")
```

Comment:
@Saransh-cpp just changed here the style code, breaking the `if` block.
just to keep consistency in general.
as pre-commit hook doesn't consider this file, we need to try to keep the consistency of this file with the other ones manually.